autopush_rs/src/client.rs

```diff
@@ -489,15 +527,30 @@ where
         flags.check = check_storage;
         flags.reset_uaid = reset_uaid;
         flags.rotate_message_table = rotate_message_table;
+
         self.webpush = Some(WebPushClient {
-            uaid: uaid,
-            flags: flags,
-            rx: rx,
-            message_month: message_month,
+            uaid,
+            flags,
+            rx,
+            message_month,
             unacked_direct_notifs: Vec::new(),
             unacked_stored_notifs: Vec::new(),
             unacked_stored_highest: None,
-            connected_at: connected_at,
+            connected_at,
+            stats: SessionStatistics {
+                uaid: String::from(uaid.hyphenated().to_string()),
```

Comment:
This is actually just `String::from(String)` so I think the `from` can be left off